set.mm

```diff
@@ -151491,7 +150711,7 @@ computer programs (as last() or lastChar()), the terminology used for
       XNXSUUNUVBUVEUVKUVIHUVCEACDUVDXNXOXPXQXBXBXR $.
   $}
 
-  $( Lemma 1 for ~ swrdccatin12OLD .  (Contributed by Alexander van der Vekens,
+  $( Lemma 1 for ~ pfxccatin12 .  (Contributed by Alexander van der Vekens,
```

Comment:
Then the label of the theorem should also be changed to ~pfxccatin12lem1, shouldn't it?

Reply:
This also holds for the following changes of this kind.